src/RadixDlt.NetworkGateway.PostgresIntegration/Services/TransactionQuerier.cs

```diff
@@ -182,10 +183,17 @@ public async Task<TransactionPageWithoutTotal> GetTransactionStream(TransactionS
             ? request.Cursor?.StateVersionBoundary ?? request.FromStateVersion
             : request.FromStateVersion;
 
-        var searchQuery = _dbContext
-            .LedgerTransactionMarkers
-            .Where(lt => lt.StateVersion <= upperStateVersion && lt.StateVersion >= (lowerStateVersion ?? lt.StateVersion))
-            .Select(lt => lt.StateVersion);
+        var baseQuery = _dbContext
+            .LedgerTransactions
+            .Where(lt => lt.StateVersion <= upperStateVersion && lt.StateVersion >= (lowerStateVersion ?? lt.StateVersion));
```

Comment:
Out of interest, have we tested this perf against a prod database?
I assume it will be a bit faster, as it just uses the PK index; but might be worth verifying

Reply:
i did run that simple query on prod and it was looking good, plan was to execute regular performance tests on dummunet with Les.